Formula/fastfec.rb

```diff
@@ -21,10 +21,7 @@ class Fastfec < Formula
 
   def install
     ENV["ZIG_SYSTEM_LINKER_HACK"] = "1"
-    args = [
-      # Use brew's pcre
-      "-Dvendored-pcre=false",
-    ]
```

Comment:
Does that mean it vendors pcre now?

Reply:
Based on https://github.com/washingtonpost/FastFEC/pull/35, sounds like they vendor PCRE sources now and don't include the option out of the box to link to a system installation of PCRE.

Reply:
Per https://github.com/washingtonpost/FastFEC/pull/46 we will be able to keep this flag value